src/main/interface/smartaudio_protocol.c

```diff
@@ -0,0 +1,187 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "platform.h"
+#include "common/crc.h"
+#include "interface/smartaudio_protocol.h"
+
+#define SMARTAUDIO_SYNC_BYTE                0xAA
+#define SMARTAUDIO_HEADER_BYTE              0x55
+#define SMARTAUDIO_GET_PITMODE_FREQ_BYTE    (1 << 14)
+#define SMARTAUDIO_SET_PITMODE_FREQ_BYTE    (1 << 15)
+#define SMARTAUDIO_FREQUENCY_MASK           0x1FFF
```

Comment:
http://www.team-blacksheep.com/tbs_smartaudio_rev08.pdf has 14 bits of frequency (0x3fff)

Reply:
Bytes 14 and 15 are reseved

Reply:
And bits 0..13 are available for frequency, the mask is 1 bit off

Reply:
I realized that right after I commented... disreagard.